### PR TITLE
prctl: add French translation

### PR DESCRIPTION
**Pull request by jay-tau**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Review by kbdharun — APPROVED** [review]

File: pages.fr/sunos/prctl.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# prctl
+
+> Obtenir ou définir les contrôles de ressources des processus, tâches et projets en cours d'exécution.
+> Plus d'information : <https://www.unix.com/man-page/sunos/1/prctl>.
+
+- Examiner les limites et les autorisations des processus:
```
Comment (kbdharun) [inline pages.fr/sunos/prctl.md:6]:
```suggestion
- Examiner les limites et les autorisations des processus :
```

File: pages.fr/sunos/prctl.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+`prctl {{PID}}`
+
+- Examiner les limites et les autorisations de processus dans un format analysable par machine:
```
Comment (kbdharun) [inline pages.fr/sunos/prctl.md:10]:
```suggestion
- Examiner les limites et les autorisations de processus dans un format analysable par machine :
```

File: pages.fr/sunos/prctl.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+`prctl -P {{PID}}`
+
+- Obtenir une limite spécifique pour un processus en cours d'exécution:
```
Comment (kbdharun) [inline pages.fr/sunos/prctl.md:14]:
```suggestion
- Obtenir une limite spécifique pour un processus en cours d'exécution :
```

**Review by kbdharun — COMMENTED** [review]
For French pages we add a space before `:`.

File: pages.fr/sunos/prctl.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
+# prctl
+
+> Obtenir ou définir les contrôles de ressources des processus, tâches et projets en cours d'exécution.
+> Plus d'information : <https://www.unix.com/man-page/sunos/1/prctl>.
+
+- Examiner les limites et les autorisations des processus :
+
+`prctl {{PID}}`
```
Comment (navarroaxel) [inline pages.fr/sunos/prctl.md:8]:
```suggestion
`prctl {{pid}}`
```

File: pages.fr/sunos/prctl.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+
+- Examiner les limites et les autorisations de processus dans un format analysable par machine :
+
+`prctl -P {{PID}}`
```
Comment (navarroaxel) [inline pages.fr/sunos/prctl.md:12]:
```suggestion
`prctl -P {{pid}}`
```

File: pages.fr/sunos/prctl.md [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+
+- Obtenir une limite spécifique pour un processus en cours d'exécution :
+
+`prctl -n process.max-file-descriptor {{PID}}`
```
Comment (navarroaxel) [inline pages.fr/sunos/prctl.md:16]:
```suggestion
`prctl -n process.max-file-descriptor {{pid}}`
```

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**